### PR TITLE
server-1943 | added filters in aws commands

### PR DESCRIPTION
**Pull request by atulsingh0**
# Description
* Add filter in AWS commands to output the information customer required

# Reasons
[SERVER-1943](https://circleci.atlassian.net/browse/SERVER-1943)

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

**Review by rosieyohannan — APPROVED** [review]

File: jekyll/_cci2/server/installation/phase-3-execution-environments.adoc [hunk 1]
```diff
@@ -46,53 +46,36 @@ Run the following command to get the cluster VPC ID (vpcId), and subnets (subnet
 +
 [source,shell]
 ----
-aws eks describe-cluster --name=<YOUR_CLUSTER_NAME>
+# Fetch VPC ID
+aws eks describe-cluster --name=<YOUR_CLUSTER_NAME> --query "cluster.resourcesVpcConfig.vpcId" -output text | xargs
```
Comment (lloydChris) [inline jekyll/_cci2/server/installation/phase-3-execution-environments.adoc:50]:
```suggestion
aws eks describe-cluster --name=<cluster-name> --region=<region> --query "cluster.resourcesVpcConfig.vpcId" --output text | xargs
```

**Review by lloydChris — CHANGES_REQUESTED** [review]
Templating should use lower case and hyphens, one might even call it "non screaming kabab case". https://circleci.com/docs/style/using-code-samples

Also, we probably don't need to prefix things with "your".  But maybe that is a nit?  Or maybe @rosieyohannan knows the rule?

File: jekyll/_cci2/server/installation/phase-3-execution-environments.adoc [hunk 2]
```diff
@@ -46,53 +46,36 @@ Run the following command to get the cluster VPC ID (vpcId), and subnets (subnet
...
+# Subnet Ids
+subnet-08922063f12541f93 subnet-03b94b6fb1e5c2a1d subnet-0540dd7b2b2ddb57e subnet-01833e1fa70aa4488 
 ----
 +
 Then, using the VPCID you just found, run the following command to get the CIDR Block for your cluster. For AWS, the DNS Server is the third IP in your CIDR block (`CidrBlock`), for example your CIDR block might be `10.100.0.0/16`, so the third IP would be `10.100.0.2`.
 +
 [source,shell]
 ----
-aws ec2 describe-vpcs --filters Name=vpc-id,Values=<YOUR_VPCID>
+aws ec2 describe-vpcs --filters Name=vpc-id,Values=<YOUR_VPCID> --query "Vpcs[].CidrBlock" --output text | xargs
```
Comment (lloydChris) [inline jekyll/_cci2/server/installation/phase-3-execution-environments.adoc:71]:
```suggestion
aws ec2 describe-vpcs --filters Name=vpc-id,Values=<YOUR_VPCID> --query "Vpcs[].CidrBlock" --output text --region=<region> | xargs
```

File: jekyll/_cci2/server/installation/phase-3-execution-environments.adoc [hunk 3]
```diff
@@ -527,7 +510,11 @@ The following command returns your VPC ID (`vpcId`) and CIDR Block (`serviceIpv4
 +
 [source,shell]
 ----
-aws eks describe-cluster --name=<your-cluster-name>
+# Fetch VPC Id
+aws eks describe-cluster --name=<YOUR_CLUSTER_NAME> --query "cluster.resourcesVpcConfig.vpcId" --output text | xargs
```
Comment (lloydChris) [inline jekyll/_cci2/server/installation/phase-3-execution-environments.adoc:514]:
```suggestion
aws eks describe-cluster --name=<YOUR_CLUSTER_NAME> --query "cluster.resourcesVpcConfig.vpcId" --output text --region=<region> | xargs
```

File: jekyll/_cci2/server/installation/phase-3-execution-environments.adoc [hunk 4]
```diff
@@ -560,7 +547,7 @@ For each https://github.com/CircleCI-Public/server-terraform/blob/main/nomad-aws
 [source,shell]
 ----
 # find CIDR block
-aws ec2 describe-subnets --subnet-ids=<NOMAD_SUBNET_ID>
+aws ec2 describe-subnets --subnet-ids=<NOMAD_SUBNET_ID> --query "Subnets[*].[SubnetId, CidrBlock]" | xargs
```
Comment (lloydChris) [inline jekyll/_cci2/server/installation/phase-3-execution-environments.adoc:550]:
```suggestion
aws ec2 describe-subnets --subnet-ids=<NOMAD_SUBNET_ID> --region=<region> --query "Subnets[*].[SubnetId, CidrBlock]" | xargs
```

**Review by lloydChris — CHANGES_REQUESTED** [review]
I think we should include some regions in the commands, otherwise many of them return empty instead of a suggestion that the user should add the region

File: jekyll/_cci2/server/installation/phase-3-execution-environments.adoc [hunk 5]
```diff
@@ -46,53 +46,40 @@ Run the following command to get the cluster VPC ID (vpcId), and subnets (subnet
...
+# Subnet Ids
+subnet-08922063f12541f93 subnet-03b94b6fb1e5c2a1d subnet-0540dd7b2b2ddb57e subnet-01833e1fa70aa4488 
 ----
 +
 Then, using the VPCID you just found, run the following command to get the CIDR Block for your cluster. For AWS, the DNS Server is the third IP in your CIDR block (`CidrBlock`), for example your CIDR block might be `10.100.0.0/16`, so the third IP would be `10.100.0.2`.
 +
 [source,shell]
 ----
-aws ec2 describe-vpcs --filters Name=vpc-id,Values=<YOUR_VPCID>
+<<<<<<< HEAD
```
Comment (lloydChris) [inline jekyll/_cci2/server/installation/phase-3-execution-environments.adoc:71]: maybe missed this on the merge?

**Review by lloydChris — CHANGES_REQUESTED** [review]

**Review by lloydChris — APPROVED** [review]

**Pull request merged**